src/nodes.ts

```diff
@@ -22,31 +23,33 @@ export type PropertyKey = Identifier | Literal;
 export type PropertyValue = AssignmentPattern | BindingIdentifier | BindingPattern | FunctionExpression;
 export type StatementListItem = Declaration | Statement;
 
+type Nullable<T> = T | null;
```

Comment:
I was also toying with the idea of e.g. `type MaybeIdentifier = Identifier | null` and use `MaybeIdentifier` at the right places. It didn't feel much worse or much better.